### ISSUE TITLE
Rule `add-constant` is not working

### ISSUE THREAD

**Issue by marek-valuch** [mgechev/revive#752]
**Describe the bug**
Arguments in rule `add-constant` doesn't work

```
/home/...: avoid magic numbers like '0', create a named constant for it (revive)
/home/...: avoid magic numbers like '1', create a named constant for it (revive)
/home/...: avoid magic numbers like '2', create a named constant for it (revive)
/home/...: string literal "" appears, at least, 3 times, create a named constant for it (revive)
/home/...: string literal "" appears, at least, 3 times, create a named constant for it (revive)
```

**To Reproduce**
Use `golang-ci` version 1.5 with following config

```
Config:
  revive:
     ...
    rules: 
      - name: add-constant
        severity: warning
        arguments:
          - maxLitCount: "10"
            allowStrs: '""'
            allowInts: "0,1,2,3"
            allowFloats: "0.0,0.,1.0,1.,2.0,2."
```
**Expected behavior**
No warnings expected

**Desktop (please complete the following information):**
 - OS: [Ubuntu 22.04]
 - Go version: 1.19.1

**Additional context**
In `golang-ci` version 1.49 these warnings doesn't occur.

**Comment by chavacava** [comment]
Hi @marek-valuch thanks for reporting the issue. 
Did you check if the problem occurs when running `revive` directly (i.e. without passing through `golangci-lint` ?

**Comment by marek-valuch** [comment]
> Hi @marek-valuch thanks for reporting the issue. Did you check if the problem occurs when running `revive` directly (i.e. without passing through `golangci-lint` ?

Yes, I tried to use same rules in `toml` config. But I can't reproduce warnings (`avoid magic numbers like...` or `string literal "" appears...`) using revive directly.

**Comment by chavacava** [comment]
@marek-valuch if the problem does not appear when running `revive` directly might indicate à problem at `golangci-lint`

**Comment by marek-valuch** [comment]
> @marek-valuch if the problem does not appear when running `revive` directly might indicate à problem at `golangci-lint`

Yes, it seems like parsing problem for `golang-ci`. Rule `add-constant` works propery when applied without arguments directly in revive. 

It is probably already reported in here: https://github.com/golangci/golangci-lint/issues/3280

Thank you!
